### DISCUSSION TITLE
Can I configure file extension for the output files?

### DISCUSSION THREAD

**Discussion by jakubmazanec** [swc-project/swc#2953] [Q&A] [answered]
Is there a way to make SWC output files with custom extension, e.g. ".mjs"? I can't simply rename the output *.js files, because the original extension is referenced in source map comment (e.g. `//# sourceMappingURL=index.js.map`). Thanks for help!

**Comment by kwonoj** [comment, marked as answer]
I may wrong, but I don't think it's supported out of the box yet.

**Reply by jakubmazanec** [reply]
Are there any plans to implement this as a feature? I think it may be useful, e.g. for creating dual ESM+CommonJS packages.

**Reply by kwonoj** [reply]
I believe esm ecosystem support should be in eventually, but need some thought how / what to do for it.
